### PR TITLE
First version of Zenbase security policy

### PR DESCRIPTION
**Pull request by viliceq**
This is a first version of our Security Policy.

The document serves multiple purposes:
- To describe already existing best practices and processes in place and pinpoint aspirational improvements
- Emphasise focus on security for all the Zenbase employees and contractors and formalise some of the processes
- An information for our existing and future partners about our policy and processes

Please review and provide feedback. Add what's missing 😊

### CODE REVIEW

**Comment by janm6k** [comment]
> All personal Google accounts are 2FA enabled

I would go much stronger than this. All work related accounts for services that support 2FA must have 2FA enabled.

File: security-policy.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Software Development
+
+The same as quality or performance, security is also everyone's job. All software engineers and everyone involved in building, running or maintaining software are equally responsible for security. Over the time and with the increasing numbers and sophistication of cyber attacks, security aspect takes bigger part of everyone's individual role.
+
+Especially for software development, the following policies and guidelines cannot ever be a complete list of checks to follow. If you see a potential loophole or vulnerability that is not covered in here, either fix it right away or at least report it! **Our security depends on everyone!**
+
+### Development process
+
+- All proprietary company source code is store in Github under company’s private repository
+- All secrets (DB passwords, API keys, etc.) are stored in a secure cloud storage only accessible to CI/CD tools and to developers on “as needed” basis
```
Comment (janm6k) [inline security-policy.md]:
```suggestion
- All secrets for production environment (DB passwords, API keys, etc.) are stored in a secure cloud storage only accessible to CI/CD tools and to developers on “as needed” basis
```

We have some secrets for demo already in git, like the Yardi test env password. For me, the convenience beats potential security implications of past colleagues having access to some sandbox passwords

File: security-policy.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Zenbase Security Policy 🔐
+
+Our security policy is not a legal document full of long hard-to-read statements. It's an easy to follow set of guidelines and rules that merely summarise practices alrady in use. When some part of this document is still in implementation phase, it's stated so.
```
Comment (loukotal) [inline security-policy.md]:
```suggestion
Our security policy is not a legal document full of long hard-to-read statements. It's an easy to follow set of guidelines and rules that merely summarise practices already in use. When some part of this document is still in implementation phase, it's stated so.
```

File: security-policy.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Software Development
+
+The same as quality or performance, security is also everyone's job. All software engineers and everyone involved in building, running or maintaining software are equally responsible for security. Over the time and with the increasing numbers and sophistication of cyber attacks, security aspect takes bigger part of everyone's individual role.
+
+Especially for software development, the following policies and guidelines cannot ever be a complete list of checks to follow. If you see a potential loophole or vulnerability that is not covered in here, either fix it right away or at least report it! **Our security depends on everyone!**
+
+### Development process
+
+- All proprietary company source code is store in Github under company’s private repository
```
Comment (loukotal) [inline security-policy.md]:
```suggestion
- All proprietary company source code is stored in Github under company’s private repository
```

**Pull request merged**